[codemod] markDynamoStrictTest batch 20

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #117609
* __->__ #117606
* #117605
* #117604

[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

zou3519 committed:
[codemod] markDynamoStrictTest batch 20

[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

[ghstack-poisoned]

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 20"


[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

[ghstack-poisoned]

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 20"


[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

[ghstack-poisoned]

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 20"


[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

[ghstack-poisoned]

bdhirsh commented on torch/testing/_internal/dynamo_test_failures.py:
```diff
@@ -7750,6 +7754,13 @@
+    "TestTensorExprPyBind.test_kernel_with_permute",
+    "TestTensorExprPyBind.test_kernel_with_scalar_inputs",
+    "TestTensorExprPyBind.test_kernel_with_t",
+    "TestTensorExprPyBind.test_kernel_with_tensor_inputs",
```
dynamo'ing TensorExpr (NNC?) code, uh oh 😛

bdhirsh replied:
Just curious, why skip and not fail - flaky?

zou3519 replied:
flaky: it failed locally for me, but passes in CI

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 20"


[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

[ghstack-poisoned]

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 20"


[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

[ghstack-poisoned]

zou3519 commented:
@pytorchbot merge -f "passed dynamo ci"

zou3519 commented:
@pytorchbot merge -f "was told I can re-merge this"

huydhn commented:
@zou3519 Some tests are failing on dynamo after this batch https://hud.pytorch.org/pytorch/pytorch/commit/308e154af5fd6388f49eabe631e7b78ca3ac9c39, just want to double check with you if fixes are coming.

zou3519 commented:
@pytorchbot revert -m "probably a crossed merge?" -c landrace

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 20"


[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

[ghstack-poisoned]

zou3519 commented:
re-landed in another PR
